docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -78,10 +78,18 @@ to be the most efficient by using the internal mechanisms.
 
 [[vector-functions]]
 ===== Functions for vector fields
+
+experimental[]
+
 These functions are used for
 for <<dense-vector,`dense_vector`>>  and
 <<sparse-vector,`sparse_vector`>> fields.
 
+NOTE: During vector functions' calculation, all matched documents are
+linearly scanned. Thus, expect the query time grow linearly 
```

Comment:
@cbuescher there was a missing `to` with this MR

 Thus, expect the query time **to** grow linearly

Reply:
Thanks. Feel free to open a PR with a correction.